ocamldoc/odoc_sig.ml

```diff
@@ -391,8 +391,13 @@ module Analyser =
               | Cstr_record l ->
                   Cstr_record (List.map (get_field env name_comment_list) l)
             in
+            let vc_name = match constructor_name with
+              | "::" ->
+                  (* The only infix constructor is always printed (::) *)
+                  "(::)"
+              | s -> s in
```

Comment:
Slight formatting quibble, it looks like `in` is usually on its own line for multi-line bindings...

Reply:
Indeed, and it's the style used right above this let-binding.
```suggestion
              | s -> s
            in
```

Reply:
Fixed.